dtfs-central-api/src/v1/controllers/utilisation-report-service/fee-record-correction/post-fee-record-correction.controller/helpers.test.ts

```diff
@@ -0,0 +1,158 @@
+import { getFormattedReportPeriodWithLongMonth, mapReasonToDisplayValue, RECORD_CORRECTION_REASON, RecordCorrectionReason } from '@ukef/dtfs2-common';
+import { formatReasonsAsBulletedListForEmail, generateFeeRecordCorrectionRequestEmailParameters, sendFeeRecordCorrectionRequestEmails } from './helpers';
+import { aBank, aReportPeriod } from '../../../../../../test-helpers';
+import { getBankById } from '../../../../../repositories/banks-repo';
+import { NotFoundError } from '../../../../../errors';
+import externalApi from '../../../../../external-api/api';
+import EMAIL_TEMPLATE_IDS from '../../../../../constants/email-template-ids';
+
+jest.mock('../../../../../repositories/banks-repo');
+jest.mock('../../../../../external-api/api');
+
+console.error = jest.fn();
+
+describe('post-fee-record-correction.controller helpers', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('formatReasonsAsBulletedListForEmail', () => {
+    it('should format reasons as a bulleted list when there is a single reasons', () => {
+      // Arrange
+      const reason = RECORD_CORRECTION_REASON.UTILISATION_INCORRECT;
+      const reasons: RecordCorrectionReason[] = [reason];
+
+      // Act
+      const result = formatReasonsAsBulletedListForEmail(reasons);
+
+      // Assert
+      const expected = `*${mapReasonToDisplayValue(reason)}`;
+      expect(result).toEqual(expected);
+    });
+
+    it('should format reasons as a bulleted list when there are multiple reasons', () => {
```

Comment:
can there ever be no reason? if there can be, then should we have a test.  ignore if not